src/parser/wakefiles.cpp

```diff
@@ -496,6 +497,8 @@ std::vector<std::string> find_workspace_wakefiles_via_manifest(const std::string
   std::stringstream in;
   in.write(reinterpret_cast<const char *>(segment.start), segment.size());
 
+  std::string prefix = (workdir == ".") ? "" : (workdir + "/");
```

Comment:
This should be "is workdir relative" not "is workdir this one specific relative path" I think.

Reply:
I think the check has to be this way, we aren't really asking is it relative, we are asking is it relative to cwd.

if `workdir` was `../workspace` and there was a file `build.wake` we should return `../workspace/build.wake` not `build.wake`.

Honestly it might just be acceptable to _always_ return `workdir + "/" + filename` as `./build.wake` is the same as `build.wake`